apps/ledger-live-mobile/src/components/OperationRow/OperationRow.test.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { Account, Operation } from "@ledgerhq/types-live";
+import { getCryptoCurrencyById } from "@ledgerhq/live-common/currencies/index";
+import BigNumber from "bignumber.js";
+
+import { render, screen } from "../../__test__/test-renderer";
+
+import OperationRow from ".";
+
+const mockedOperation: Operation = {
+  accountId: "js:1:ethereum:0xaccount",
+  id: "js:2:ethereum:0xa22CA840265d3C5CB1846e419B14c6a6CdD06FAB:",
+  hash: "eth_one",
+  type: "OUT",
+  value: new BigNumber(6453567),
+  fee: new BigNumber(4890),
+  senders: ["0xsender"],
+  recipients: ["0xrecipient"],
+  blockHeight: null,
+  blockHash: null,
+  transactionSequenceNumber: undefined,
+  date: new Date(),
+  extra: {},
+};
+
+const ethereum = getCryptoCurrencyById("ethereum");
+
+const mockedAccount: Account = {
```

Comment:
we have mock accounts for unit tests. They are defined in /ledger-live/libs/ledger-live-common/src/mock/account.ts
should we use it instead?